docs/CLI.md

```diff
@@ -0,0 +1,87 @@
+# OPAT CLI
```

Comment:
Love the docs! I think we may ultimately want to move some of these sections to the README, but this is fine for now

Reply:
Agreed, I did not want to clutter the README right now hence the separate document.